Sources/OpenAPIRuntime/Deprecated/Deprecated_RedactedHeaderFields.swift

```diff
@@ -14,7 +14,7 @@
 #if canImport(Darwin)
 import Foundation
 #else
-@preconcurrency import Foundation
+@preconcurrency import Foundation.NSLock
```

Comment:
```suggestion
@preconcurrency import class Foundation.NSLock
```

Credit goes to CI for discovering this.

Reply:
You can locally test this as well with e.g. `docker-compose -f docker/docker-compose.yaml -f docker/docker-compose.2204.58.yaml run test`